src/structures/VoiceState.js

```diff
@@ -152,10 +152,10 @@ class VoiceState extends Base {
   }
 
   /**
-   * Moves the member to a different channel, or kick them from the one they're in.
-   * @param {ChannelResolvable|null} [channel] Channel to move the member to, or `null` if you want to kick them from
-   * voice
-   * @param {string} [reason] Reason for moving member to another channel or kicking
+   * Moves the member to a different channel, or disconnects them from the one they're in.
+   * @param {ChannelResolvable|null} [channel] Channel to move the member to, or `null` if you want to disconnect them
+   * from voice. Requires the `MOVE_MEMBERS` permission.
```

Comment:
I'm not exactly sure if the permission requirement should be added for this one.
No other method has required permission listed in its description. Additionally, that adds one more thing to watch for upstream changes, no matter how likely that is to change.